cmake/std/atdm/cts1/environment.sh

```diff
@@ -0,0 +1,29 @@
+################################################################################
+#
+# Set up env on cts1 for ATMD builds of Trilinos
+#
+# This source script gets the settings from the ATDM_CONFIG_BUILD_NAME var.
+#
+################################################################################
+
+# Chama and cts1 jobs all use the same environmnet changes to the
+# sourced script below will impact jobs on both of those
+# machines. please be mindful of this when making changes
+
+if [ "$ATDM_CONFIG_KOKKOS_ARCH" == "DEFAULT" ] ; then
+  export ATDM_CONFIG_KOKKOS_ARCH=BDW  
+fi
+
+if [ "$ATDM_CONFIG_KOKKOS_ARCH" != "BDW" ] ; then
+  echo "***"
+  echo "*** ERROR: KOKKOS_ARCH=$ATDM_CONFIG_KOKKOS_ARCH is not a valid option on this system."
+  echo "*** '$ATDM_CONFIG_KOKKOS_ARCH' appears in $ATDM_CONFIG_BUILD_NAME which then sets the KOKKOS_ARCH"
+  echo "*** on cts1 'BDW' is the only valid KOKKOS_ARCH. If no KOKKOS_ARCH is specified then"
+  echo "*** 'BDW' will be used by default"
+  echo "***"
+  return
+fi
+
+export ATDM_CONFIG_ENABLE_SPARC_SETTINGS=ON
```

Comment:
Does this work?  I don't see any changes to `common/toss3/environment.sh` that would allow this to configure.

Also, I suspect that we might need to temporarily duplicate what is in `common/toss3/environment.sh` in order to get stuff working with the SPARC TPLs instead of the TPLs it is using now in the devpack module (that we will not be using anymore).